backup_and_restore/restore-acs.adoc

```diff
@@ -8,10 +8,53 @@ toc::[]
+It allows you to restore and recover the Central database to its earlier functional state.
+. Restore Central from the Central deployment backup. Use this if you are migrating Central to another cluster or namespace. This option restores the configurations of your Central installation.
 
-include::modules/restore-acs-roxctl-admin-pass.adoc[leveloffset=+1]
+[id="restore-central-db-roxctl"]
+== Restoring Central database by using the `roxctl` CLI
+You can use the `roxctl` CLI to restore {product-title} by using the `restore` command. You require an API token or your administrator password to run this command.
 
-include::modules/resume-acs-restore.adoc[leveloffset=+1]
+include::modules/restore-acs-roxctl-api.adoc[leveloffset=+2]
+
+include::modules/restore-acs-roxctl-admin-pass.adoc[leveloffset=+2]
+
+include::modules/resume-acs-restore.adoc[leveloffset=+2]
+
+[id="restore-central-deployment-roxctl"]
+== Restoring Central deployment using the `roxctl` CLI
+You can restore your Central deployment to its original configuration by using the backups you made.
+To successfully restore, you need Central custom resource, `central-tls` and the administrator password.
+`central-tls` includes the certificates for authenticating with Secured clusters and signing API tokens.
+
+You must first restore certificates by using the `roxctl` CLI, and then restore the Central deployment by running the Central installation scripts.
+
+include::modules/restore-cert-roxctl.adoc[leveloffset=+2]
+include::modules/install-central-roxctl.adoc[leveloffset=+2]
+
+include::modules/restore-deployment-config-operator.adoc[leveloffset=+1]
+
+include::modules/restore-deployment-config-helm.adoc[leveloffset=+1]
+
+[id="restore-central-another-cluster"]
+== Restoring central to another cluster or namespace
+You can use the backups of the {product-title-short} Central database and the deployment to restore Central to another cluster or namespace.
+
+The following list provides a high-level overview of installation steps:
+
+. Depending upon your installation method, you must first restore Central deployment by following the instruction in the following topics:
++
+[IMPORTANT]
+====
+* Make sure to use the backed-up Central certificates so that secured clusters and API tokens issued by the old Central instance remain valid.
+* If you are deploying to another namespace, you must change the namespace in backed-up resources or commands.
+====
+
+** xref:../backup_and_restore/restore-acs.adoc#restore-central-deployment-roxctl[Restoring Central deployment using the `roxctl` CLI]
+** xref:../backup_and_restore/restore-acs.adoc#restore-deployment-config-operator_restore-acs[Restore Central deployment using the {product-title-short} Operator]
+** xref:../backup_and_restore/restore-acs.adoc#restore-deployment-config-helm_restore-acs[Restore Central deployment using Helm]
+. Follow the section to Restore Central data TODO
```

Comment:
@johannes94 Doesn't the previously mentioned sections cover this already?

Reply:
I think it's not, the whole path is in this sequence:

1. Restore deployment (helm or operator method)
2. Restore data (with roxctl no matter which installation method)
3. Restore routing configuration